docs/change-controlled-docs.rst

```diff
@@ -0,0 +1,293 @@
+.. _writing-ccd:
+
+#######################################
+Writing change-controlled documentation
+#######################################
+
+Change-controlled documents are documents that require approval from a Change Control Board (CCB) before they can be released.
```

Comment:
We probably should have the note at the top declaring that this page is change-controlled.